src/main/java/AlexLink/HomeWork/IO/HomeWork12/Task1.java

```diff
@@ -0,0 +1,46 @@
+package AlexLink.HomeWork.IO.HomeWork12;
+
+//1)� �������� ����� ��������� �����, ������ ����� �� ����� ������. �����
+//������� ��������� ������ ������� ����, ������� ����� ��������� � ����
+//������ ����������
+
+import java.io.*;
+
+public class Task1 {
+    public static void main(String[] args) {
+        String text = "far alone oko potop here";
+        File file = new File("polindromes.txt");
```

Comment:
According to requirements, each word must be on new line. I see that in this file all words in on line.

Reply:
fixed